apps/engage-hub-service/src/routines/dto/create-routine.dto.ts

```diff
@@ -35,4 +35,8 @@ export class CreateRoutineDto {
   @IsNotEmpty()
   @IsString()
   description: string;
+
+  @IsOptional()
+  @IsUUID()
+  userId: UUID | null; 
```

Comment:
Instead to create column named `userId` use `creatorId` one

Reply:
done